src/renderer/components/watch-video-comments/watch-video-comments.js

```diff
@@ -204,16 +204,13 @@ export default Vue.extend({
         comment.authorThumb = comment.authorThumb[0].url
         comment.replies = []
         comment.dataType = 'local'
-        this.toLocalePublicationString({
+        comment.time = toLocalePublicationString({
           publishText: (comment.time + ' ago'),
           isLive: false,
           isUpcoming: false,
           isRSS: false
```

Comment:
As you have added `false` as the default value for those 3 parameters, you don't need to pass them in here.